compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/util/Constants.java

```diff
@@ -40,4 +40,5 @@ private Constants() {
 
     public static final int INIT_METHOD_SPLIT_SIZE = 50;
 
+    public static final String UNICODE_REGEX = "\\\\u\\{([a-fA-F0-9]*)\\}";
```

Comment:
Lexer rule is defined as `HexDigit+`. But here, the regex is defined as `[a-fA-F0-9]*`. I think this can be changed to `[a-fA-F0-9]+`. WDYT?

Reply:
Done